user-operator/src/main/java/io/strimzi/operator/user/model/KafkaUserModel.java

```diff
@@ -95,17 +98,19 @@ protected KafkaUserModel(String namespace, String name, Labels labels) {
      * @param clientsCaCert The clients CA certificate Secret.
      * @param clientsCaKey The clients CA key Secret.
      * @param userSecret Kubernetes secret with existing user certificate.
+     * @param secretPrefix
```

Comment:
missing JavaDoc description about this parameter

Reply:
Thank you for caching this.